clayui.com/content/docs/get-started/using-clay-in-jsps.md

```diff
@@ -0,0 +1,70 @@
+---
+title: 'Using Clay in JSPs'
+order: 6
+---
+
+<div class="nav-toc-absolute">
+<div class="nav-toc">
+
+-   [Setting up your module](#setting-up-your-module)
+-   [Using Clay taglib in your JSPs](#using-clay-taglib-in-your-jsps)
+-   [Clay components available in JSPs](#clay-components-available-in-jsps)
+
+</div>
+</div>
+
+Clay in combination with DXP provides a set of tags for creating Clay components in your JSP files.
+
+## Setting up your module
```

Comment:
I have a little bit of concern about it, @mwilliams2014 has done a lot of work documenting the use of JSP for Clay v2 components and I am concerned that this may become more difficult to maintain because the code is on Liferay and that we may have duplicate information, in theory, the next updates should "follow" the component APIs so there would be no problem but I still wonder if we should just link to the Liferay developer documentation.

Another concern is that the JSP comes with Liferay DXP and depending on the version that the person is using the DXP it may be that the JSP or API he is seeing here is not available or the APIs are only available in the latest version of DXP or on the master itself.

So what I want to say is that we should be concerned with this before going ahead and aligning it if we want these documents to be only for internal DXP developers and keep only updated according to the master or we would have to follow the DXP version and update only after releases. If it is for internal developers, we must make this clear and also the person if they are looking for a specific version or if it is a customer or partner forwarding to the Liferay DXP documentation.

cc @jbalsas @wincent

Reply:
Thanks for the mention @matuzalemsteles . We definitely don't want to duplicate information in both places. If you have documentation that's for our customers, that relates directly to its use in Liferay, please draft it and send it to me for review, so we can publish it with/improve our current documentation. If it's meant for internal usage/is clay specific (not relating to Liferay) then it makes more sense to publish that information with Clay's documentation.

As far as Liferay features that are DXP only, we tend to note that above the documentation along with the version (i.e. DXP 7.3 SP2+), so users are aware of that, and update just before/after releases as you mentioned. Thanks!